distributed/nanny.py

```diff
@@ -314,8 +314,20 @@ def run_worker_fork(q, scheduler_addr, ncores, nanny_port,
     """
     Create a worker by forking.
     """
+    import os
+    import random
     from distributed import Worker  # pragma: no cover
+    from distributed.metrics import time
     from tornado.ioloop import IOLoop  # pragma: no cover
+    random.seed((time(), os.getpid()))
```

Comment:
Using `random.seed()` without any argument makes a best effort to seed with enough high-quality entropy.